beam/hooks.py

```diff
@@ -516,3 +516,88 @@
 		],
 	},
 }
+
+
+beam_mobile = {
+	"components": {
+		"Home": "./beam/beam/www/beam/pages/Home.vue",
+		"Workstation": "./beam/beam/www/beam/pages/Workstation.vue",
+		"WorkOrder": "./beam/beam/www/beam/pages/WorkOrder.vue",
+		"Receive": "./beam/beam/www/beam/pages/Receive.vue",
+		"Ship": "./beam/beam/www/beam/pages/Ship.vue",
+		"Demand": "./beam/beam/www/beam/pages/Demand.vue",
+		"Manufacture": "./beam/beam/www/beam/pages/Manufacture.vue",
+		"Move": "./beam/beam/www/beam/pages/Move.vue",
+		"Repack": "./beam/beam/www/beam/pages/Repack.vue",
+		"JobCard": "./beam/beam/www/beam/pages/JobCard.vue",
+		"Operation": "./beam/beam/www/beam/pages/Operation.vue",
```

Comment:
@agritheory Since these paths will be analyzed statically during build-time and must still remain relative, I think we should document somewhere what is expected in these hooks.

The current rule is: path from the app's root level-folder.

What do you think?

Reply:
Yes, I think that makes sense, both from a DX and documentation perspective